ocean_lib/ocean/ocean_assets.py

```diff
@@ -96,6 +99,38 @@ def _process_service_descriptors(
         if access_service_descriptor:
             _service_descriptors.append(access_service_descriptor)
         if compute_service_descriptor:
+            if trusted_algorithms:
+                trusted_algo_list = []
```

Comment:
I would separate the creation of trusted_algo_list into a different function. It seems there's too much indentation here.